### DISCUSSION TITLE
Is it possible to make a Xiaomi WSDCGQ11LM (humidity sensor) less sensitive, to safe battery?

### DISCUSSION THREAD

**Discussion by Suzabi** [Koenkk/zigbee2mqtt#14048] [Devices]
Hello all,

I have three humidity sensors, which I would like to make less sensitive to temperature and humidity changes, or make them send data less frequently, since the battery is going down really quick.

Is it possible to change this, and how can I do it?

**Comment by lux73** [comment]
define what is really quick for you...?  i'm using 6 of them and my Battery's at least survive 1 Year...  mostly longer!

BUT i'm not using the cheapest Battery's one i could get - i'm using Panasonic Brand. Had terribly Experience with no-name CR2032

btw its not possibly to affect this behavior - the Aqara Sensors send always Data if the Temp/Humidity/Pressure changes

**Reply by Suzabi** [reply]
I got some batteries for the supplier where I bought them. Accordingly, to the data I got from Home Assistant, they lost around 50% in 1 Month, but it seems like it is not linear. The lost more per day in the first week, then in the second, and so on.
![image](https://user-images.githubusercontent.com/59054542/190926157-35905c6d-f996-4b77-a987-546ff62a2687.png)
